prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -59,10 +59,5 @@ def workbench_scenarios():
         """A canned scenario for display in the workbench."""
         return [
             ("{{cookiecutter.class_name}}",
-             """<vertical_demo>
-                <{{cookiecutter.short_name|lower}}/>
-                <{{cookiecutter.short_name|lower}}/>
-                <{{cookiecutter.short_name|lower}}/>
-                </vertical_demo>
-             """),
+            """<{{cookiecutter.short_name|lower}}/>"""),
```

Comment:
I'd like to be sure that the default scenario always render more than
one instance of an XBlock. An all too common web development issue is to
design/develop with just a single "widget" instance on the page. This
leads to inevitable issues between IDs, classes, and other selectors.
Problems range from CSS/JS selectors to issues with data/form
submission.

It doesn't need to be 3, but by rendering at least 2 XBlocks, we ensure
that developers are forced to work from a multiple-XBlock-per-page
perspective.